news/2022-09-16-project-loved-september-2022.md

```diff
@@ -0,0 +1,341 @@
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/973421.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Hatsuki Yura - Kaguyahime](https://osu.ppy.sh/beatmapsets/973421#fruits) by [Kukkai](https://osu.ppy.sh/users/7811952)\
+154 BPM, 4:21 | 6.18★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+*Kaguyahime* is a mapset which has been in graveyard since October 2019, which is a shame because it contains a really clean and well done catch difficulty mapped by Kukkai. Compared to last month's selection of maps this one is way easier to play but still requires a good amount of consistency and speed to be able to get a full combo. It has a lot of fast-paced patterns which may catch you off guard if you aren't ready for them, especially the later half of the map, which could mark the end of a lot of good runs, because that's where it gets increasingly difficult. All in all this is a fun and neat practice map for aspiring players who want to challenge themselves, but also playground for top players to see how they can do with HR.
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1134669.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Hommarju & Dollscythe - Flashes (Hommarju Remix)](https://osu.ppy.sh/beatmapsets/1134669#fruits) by [Fauzan 2000](https://osu.ppy.sh/users/9317817)\
+175 BPM, 2:35 | 8.11★\
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+WUB WUB WUB and other things! You heard that right, we are throwing *Flashes* by *Fauzan 2000* into the loved section; another mapper with their first entry into loved! This is a fairly difficult map, containing many jumps and moderately large sliders that accentuate the aforementioned "WUB" quite well. It is just really satisfactory to indulge yourself in the song — perhaps even putting your audio system to daringly loud volumes and enjoying the intense electronics making your fingers dance. Want to throw in a mod or two? That should be doable for the upper echelon of players, without busting the entire experience.
+
+Sooooo, flash forward to this next paragraph: was I able to convince you to vote for this map? Give it a spin either way!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1205520.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Tomoya Ohtani - Crisis City (SoInhumane Remix)](https://osu.ppy.sh/beatmapsets/1205520#fruits) by [Sutavv](https://osu.ppy.sh/users/5320583)\
+181 BPM, 4:07 | 11.21★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Hyperwalk enjoyers, rejoice! This week we have another wild map that is filled to the brink with hyperwalks everywhere. This map very much gives me the vibes of [Obsidian](https://osu.ppy.sh/beatmapsets/1130748#fruits) which was Loved about a year ago, calm patterns here and there followed by immediate hyperwalks and insane patterns.
```

Comment:
```suggestion
Hyperwalk enjoyers, rejoice! This week we have another wild map that is filled to the brim with hyperwalks everywhere. This map very much gives me the vibes of [Obsidian](https://osu.ppy.sh/beatmapsets/1130748#fruits) which was Loved about a year ago — calm patterns here and there followed by immediate hyperwalks and insane patterns.
```

Reply:
forgot to update the PR but this had a different edit on loved.sh that also resolves, i think